### DISCUSSION TITLE
waltid-wallet-api not working

### DISCUSSION THREAD

**Discussion by ChangoBuitrago** [walt-id/.github#78] [Help] [answered]
Hi there,
Below is the entire Docker Compose startup log, along with a reference to the previous Discord chat where attempts were made to solve it.
Thank you!

- discord_support_chat: https://discord.com/channels/950664482877763656/1221079187784339467
- docker-compose_startup_log: [whole_docker-compose_startup.log](https://github.com/walt-id/.github/files/14764723/whole_docker-compose_startup.log)

NOTE: if you run it as described in the [README.md](https://github.com/walt-id/waltid-identity/blob/main/waltid-wallet-api/README.md) file (standalone container), meaning:
```
docker run -p 7001:7001 -itv $(pwd)/waltid-wallet-api/config:/waltid-wallet-api/config -itv $(pwd)/waltid-wallet-api/data:/waltid-wallet-api/data -t waltid/wallet-api
```
It does work (although it continues to show those configuration errors); here is the log:
- standalone_log: [standalone-container.log](https://github.com/walt-id/.github/files/14764809/standalone-container.log)

**Comment by alegomes** [comment, marked as answer]
Issue fixed on [PR #265](https://github.com/walt-id/waltid-identity/pull/265)

**Comment by ChangoBuitrago** [comment]
Thanks to the Walt.id team and the special support of Alexandre Gomes.
